packages/web-components/src/components/va-promo-banner/va-promo-banner.css

```diff
@@ -19,8 +21,8 @@
 .va-promo-banner__icon {
   background-color: var(--vads-color-white);
   border-radius: 50%;
-  height: 4.2rem;
-  width: 4.2rem;
+  height: 42px;
+  width: 42px;
```

Comment:
I think it was a typo how the values were set here so this updates it to match what's in [Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=2941-2265&t=HzGcE4nkWLtVsfi1-0):

![Screenshot 2024-06-11 at 10 25 42 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/2c4162c7-c9c8-4e8c-8fbb-12c4f579af0e)

![Screenshot 2024-06-11 at 10 25 48 AM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/c349b894-357f-436f-9d43-9e49f1933c9d)